### DISCUSSION TITLE
Support for resizing and cropping?

### DISCUSSION THREAD

**Discussion by gregbenz** [google/libultrahdr#37] [Q&A] [answered]
Does libultrahdr support basic reprocessing of an existing gain map? Specifically, can it take an Ultra HDR JPG and resize and/or crop it to generate a new derivative JPG gain map? This would of course me important for derivative content in media libraries (thumbnails, etc).

**Comment by arifdikic** [comment, marked as answer]
Hi Greg, this is on our roadmap, will be there by March or so.

**Reply by gregbenz** [reply]
That's great, thank you for all you're doing to support HDR photography!
